pkg/test/echo/common/util.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 const (
-	ConnectionTimeout     = 1 * time.Minute
+	ConnectionTimeout     = time.Second * 2
```

Comment:
In negative test cases, we may see a connection timeout. In these cases, the tests take 1 minute to pass (!). 2s seems reasonable here, given that we have retries higher up in the code

Reply:
Nice. Can you add this comment to the code so it's easier to track.